### PR TITLE
✨  Added member impersonation

### PR DESCRIPTION
**Pull request by gargol**
refs https://github.com/TryGhost/Ghost/pull/11607

@kevinansfield single thing that I have in question for this PR is the use of adapter on member model 
2b98a3d, instead of a separate model for `signin_url` resource. Do you think this is clean approach or should be done differently?

### CODE REVIEW

**Comment by gargol** [comment]
@kevinansfield refactored the adapter call to a straight ajax in the member model. Will be merging this later today unless there are any other objections :wink:

File: app/components/modal-impersonate-member.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    store: service(),
+
+    classNames: 'modal-impersonate-member',
+
+    signinUrl: null,
+    member: alias('model'),
+
+    init() {
+        this._super(...arguments);
+    },
```
Comment (kevinansfield) [inline app/components/modal-impersonate-member.js]: This isn't needed 🙂

File: app/components/modal-impersonate-member.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    didInsertElement() {
+        this._super(...arguments);
+
+        this._signinUrlUpdateTask.perform();
+    },
+
+    actions: {},
```
Comment (kevinansfield) [inline app/components/modal-impersonate-member.js]: Nor this

File: app/components/modal-impersonate-member.js (hunk 1 above)
Comment (gargol) [reply app/components/modal-impersonate-member.js]: Corrected both. Thanks for catching!

**Pull request merged**